lib/rubocop/result_cache.rb

```diff
@@ -97,7 +97,13 @@ def load
 
     def save(offenses)
       dir = File.dirname(@path)
-      FileUtils.mkdir_p(dir)
+
+      begin
+        FileUtils.mkdir_p(dir)
+      rescue Errno::EACCES
+        return
```

Comment:
Shouldn't we at least show some warning to the users?

Reply:
@bbatsov Yeah, I agree with you. What about this?
```ruby
      begin
        FileUtils.mkdir_p(dir)
      rescue Errno::EACCES => e
        warn "Can't create cache directory! #{e.message}"
        return
      end
```

=> `Can't create cache directory! Permission denied @ dir_s_mkdir - /permission_denied_dir`

Reply:
+1 for a warning message. Perhaps also an indication how we handle it?

```
Couldn't create cache directory. Continuing without cache.
  Pemission denied @ dir_s_mkdir - /permission_denied_dir
```